### DISCUSSION TITLE
Don’t fret about students using ChatGPT to cheat – AI is a bigger threat to educational equality

### DISCUSSION THREAD

**Discussion by Aaron25564448** [1101-GS4719-C/Python-Community-GS4719-C#1229] [General]
https://theconversation.com/dont-fret-about-students-using-chatgpt-to-cheat-ai-is-a-bigger-threat-to-educational-equality-202842
大家總是說AI的進化能為人們大幅提升工作效率，但這種現象是否只局限於那些「有資源的人」呢？學校總是擔心學生利用AI作弊或是完成報告、作業，抑或是論文。然而，從另一方面檢視這種狀況，資源不足，也就是有所謂「資訊落差」的人所能獲得的教育資源將有可能與有資源者越差越多。舉例來說，Chat GPT會員一個月須花費20美元，這般支出對其他國家的人有可能成為另一筆負擔；此外，許多語言也可能因此消失。即使Chat GPT能輸出多種語言，但並未有部分國家、部落獨有的傳統語言，造成文化消失速度增加。我個人是非常認同這篇文章的看法。畢竟在享受科技的同時，人們常會認為這是理所當然的，產生一種「既然到了我手上，白用白不用」的心態，卻常忘記並不是每個人都有辦法入手這些工具。只能希望科技的發展能夠盡快減低資訊上的貧富差距。

**Comment by DuxtinTai** [comment]
為了解決不平等的問題，我認為開放基礎的人工智慧工具供大眾免費使用是必要的。當我們開放基礎的人工智慧工具供大眾免費使用時，可以讓更多人擁有運用人工智慧的能力，不僅能夠增進個人能力，還能夠促進社會的發展。對於那些資源匱乏的人來說，他們有了使用這些工具的機會，可以從中學習到新知識和技能，提高自己的競爭力。而對於那些已經掌握了基礎知識的人，這些工具可以幫助他們更加高效地完成工作，節省時間和精力。此外，開放基礎的人工智慧工具也有助於促進跨學科的合作，使不同領域的人士可以共同使用這些工具，從而產生出更多的創新和解決方案。

**Comment by KevinCheng42** [comment]
我覺得這也反映了各國家文化方面的不平等，從古至今，從編寫書籍一直到現在3C產品及網路資源的使用，商家或企業為了利益最大化，會挑選較多人使用的語言提供服務，因此，常常忽略部分國家、部落獨有的傳統語言，並且，這不是短時間可以解決的問題。

**Comment by YoYooHuang** [comment]
關於學校擔心學生利用chatgpt作弊這件事情，我個人認為重點並不在於要防止學生作弊，而是要讓學生學會如何正確地使用chatgpt。最明顯的例子就是，大家來到同一所大學，但都來自不同的地方，所有人的先天背景都大不相同，所擁有的資源和先天知識也大相逕庭。這時，若能善用AI工具，那些落後的同學也能加快腳步，跟上其他同學的實力。只要出發點是吸收知識，增強實力，chatgpt絕對是一款好用的學習工具。

**Comment by curtis1017** [comment]
我覺得我們將重心不該放在防止學生作弊，而是要讓學生學會如何正確地使用chatgpt。一種新的科技出現，帶給我們許多便利，但我們不能放棄新技術而守舊，這對於這個科技時代不會造成好處。

**Reply by chia-kai-chang** [reply]
沒錯，我一直是抱持著這種觀點的，工具會一直演進，我們可以把節省下來的時間去思考更多有創意的事情，或是更認真地體驗這個世界。
